api/blocks/blocks.go

```diff
@@ -47,6 +54,16 @@ func (b *Blocks) handleGetBlock(w http.ResponseWriter, req *http.Request) error
 		return err
 	}
 
+	if raw == "true" {
+		rlpEncodedSummary, err := rlp.EncodeToBytes(summary)
```

Comment:
```suggestion
		rlpEncoded, err := rlp.EncodeToBytes(summary.Header)
```
Here we should use Header.